client/src/app/fridge/fridge.component.ts

```diff
@@ -1,16 +1,19 @@
 import { Component, OnInit } from '@angular/core';
-import { FridgeService } from '@services';
+import { Ingredient, FridgeService } from '@services';
 
 @Component({
   selector: 'app-fridge-component',
   templateUrl: './fridge.component.html',
   styleUrls:['./fridge.component.scss']
 })
 export class FridgeComponent implements OnInit{
-  constructor(private fridgeService: FridgeService) {}
+  constructor(private fridgeService: FridgeService) {
+    this.getIngredients();
+  }
 
   addedIngredients: Set<string> = this.fridgeService.getLocallyStoredIngredients();
```

Comment:
Variables should appear before the constructor. The order goes variables > constructor > methods, and public always before private.